lib/endpoint/find-parameters.js

```diff
@@ -227,3 +228,15 @@ function makeRelativeLinksAbsolute (state) {
   })
   return state
 }
+
+function replaceTimeNowDefault (state) {
+  state.results.forEach(output => {
+    for (let i = 0; i < output.params.length; i++) {
+      if (output.params[i].default === 'Time.now') {
+        output.params[i].default = output.params[i].default.replace('Time.now', '<current date/time>')
```

Comment:
I think you can simplify this line. No need to use `.replace()`, you can just set the new `.default` to '<current date/time>'